contracts/ProjectRegistry.sol

```diff
@@ -126,7 +126,11 @@ contract ProjectRegistry is Initializable, OwnableUpgradeable, ERC721Upgradeable
     }
 
     function supportsInterface(bytes4 interfaceId) public view override(ERC721Upgradeable, ERC721EnumerableUpgradeable) returns (bool) {
-        return super.supportsInterface(interfaceId);
+        return
```

Comment:
Use `super.supportsInterface(interfaceId)` to check the extension interface should be alright

```
return interfaceId == type(IProjectRegistry).interfaceId || super.supportsInterface(interfaceId);
```

Reply:
From my investigation, `super` is unpredictable, when you have multiple inheritance.
Of course, we don't have a use case when and who will call `supportsInterface()`, so this is unurgent.

Reply:
https://docs.soliditylang.org/en/latest/contracts.html#multiple-inheritance-and-linearization
In Solidity, it should follow the rule: `right -> left: higher -> low`,  as the right one `ERC721EnumerableUpgradeable` also inherit from `ERC721Upgradeable`, only one super call should be fine.